README.md

```diff
@@ -83,6 +84,42 @@ GLHF! <3
 
 GLHF! <3
 
+## Tantangan
+
+Sepanjang bulan November, Anda dapat membuat sebuah permainan *bebas* dengan bertemakan **peretasan, modifikasi dan/atau penggandaan**.
+
+Apa yang kita maksud dengan **bebas** berdasarkan pada peretasan, modifikasi dan/atau penggandaan? Maksud kami, benar-benar bebas, secara harfiah tidak ada syarat dan ketentuan. Contohnya seperti:
+
+* game berlari tanpa ujung di mana anda menebang (*hack*) pohon biner di sepanjang jalur anda dengan kapak piksel
+* game versi *modern* dari permainan klasik, misalnya sebuah roguelike yang diatur di dalam dunia 3D atau dunia VR, atau
+* Sebuah permainan realitas tertambah (*augmented reality*) yang membawa gurita/kucing ke dalam dunia nyata.
+
+Tunjukan kreatifitas anda. Anda dapat bekerja sendiri atau dengan tim dan membangun untuk berbagai platform ataupun perangkat. Anda dianjurkan, tetapi tidak diharuskan untuk menggunakan mesin permainan dari sumber terbuka dan berbagai pustaka.
+
+Kami akan tetap memantau beberapa blog GitHub favorit kami, dan dunia dapat menikmati (dan bahkan mungkin berkontribusi dan/atau untuk belajar) dengan kreasi anda.
+
+## Bagaimana cara berpartisipasi
+
+* [Daftarkan akun][github-signup] jika anda belum punya.
+* *Fork* [repositori ini][game-off-repo] untuk akun pribadi (atau untuk sebuah [akun organisasi gratis][github-signup-org]).
+* *Clone* repositori pada komputer anda dan buat permainan anda.
+* *Push* kode sumber(source code) permainan anda untuk repositori hasil *fork* anda sebelum tanggal 1 Desember.
+* Perbaharui berkas `README.md` untuk memasukkan penjelasan dari permainan anda, bagaimana/dimana untuk memainkan/mengunduhnya, bagaimana cara build/compile permainan anda, ketergantungan apa yang dimiliki, dan lain-lain.
```

Comment:
I think like a fork and clone no need to translate because it's a term used GitHub